src/lib/data/emojis.json

```diff
@@ -0,0 +1,1879 @@
+{
+	"+1": "https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png?v8",
```

Comment:
Dont hyperlink github. They will gut us

Reply:
Now using key instead, no call to github ?

Reply:
> Now using key instead, no call to github ?

`https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png?v8`

Images are still on github

Reply:
> Now using key instead, no call to github ?

`https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png?v8`
Images are still stored on github